radixdlt/src/test/java/com/radixdlt/network/messaging/MessageDispatcherTest.java

```diff
@@ -168,21 +162,41 @@ public void receiveExpiredMessage() {
         verify(counters, times(1)).increment(CounterType.MESSAGES_INBOUND_DISCARDED);
     }
 
+    @Test
+    public void receiveMessageFromBannedPeer() {
+    	// Banned for a long time
+    	peer1.setBan("Test", System.currentTimeMillis() + 86_400_000L);
+    	when(this.addressBook.peer(any(EUID.class))).thenReturn(Optional.of(peer1));
+    	RadixSystem system = mock(RadixSystem.class);
+    	doReturn(EUID.TWO).when(system).getNID();
+    	doReturn(Radix.AGENT_VERSION).when(system).getAgentVersion();
```

Comment:
weird tabbing

Reply:
Actually, this whole file seems to use The Correct Indentation Method, rather than tabs.  Will uncorrect it.